pkg/ovsdb/dbserver.go

```diff
@@ -259,3 +270,51 @@ func (con *DBServer) GetMarshaled(prefix string, columns []interface{}) (*[]map[
 	}
 	return &values, nil
 }
+
+func (con *DBServer) GetSchema(name string) (string, bool) {
+	return con.schemas[name], true
+}
+
+func (con *DBServer) GetUUID() string {
+	return con.uuid
+}
+
+type DBServerMock struct {
+}
+
+func NewDBServerMock() (DBServerInterface, error) {
```

Comment:
I'd prefer to put the Mock into a separate file

Reply:
this very C++ or Java but is not the norm in go, as is creates many many very small files (each with overhead of header includes etc.), ... I can show many examples to support this claim in leading go projects